tarodb/interfaces.go

```diff
@@ -113,13 +113,15 @@ func (t *TransactionExecutor[Q]) ExecTx(ctx context.Context,
 	}()
 
 	if err := txBody(t.createQuery(tx)); err != nil {
+		err = MapSQLError(err)
```

Comment:
nit: can just do `return MapSQLError(err)` here and below.

Reply:
And below I see :+1: